rpc/block_genconfig.c

```diff
@@ -22,7 +22,7 @@ getTpgObj(char *block, MetaInfo *info, blockGenConfigCli *blk, char *portal, int
   uuid_t uuid;
   char alias[UUID_BUF_SIZE];
   char alias_10[11] = {'\0', };
-  char lun_so[256] = {'\0', };
+  char lun_so[384] = {'\0', };
```

Comment:
@lxbsz how did you calculate this ? why not keep it 512 ?

Reply:
The 256 will be the length of the "char *block", which comes from the d_name, and the 128 will be "/backstores/user/" string.

I am okay with the 512, 384 here just wanted to make sure the array size which hosted in the C stack as small as possible.

Reply:
Okay. If you want to keep the stack small then 128 is also bigger for fixed string here.

strlen("/backstores/user/") = 17

17 + 256, will be good enough. But I will leave it to you, if you want to keep it as it is. In any case making a small comment about the calculation might help the readers.

Thanks!

Reply:
Yeah, the comment here make sense.
Just add 32 extra bytes instead.
Thanks.